pkg/occlient/occlient.go

```diff
@@ -107,6 +118,63 @@ func runOcComamnd(command *OcCommand) ([]byte, error) {
 	return output, nil
 }
 
+func isLoggedIn() bool {
+	ocpath, err := getOcBinary()
+	if err != nil {
+		log.Debug(errors.Wrap(err, "unable to find oc binary"))
+		return false
+	}
+
+	cmd := exec.Command(ocpath, "whoami")
+	output, err := cmd.CombinedOutput()
+	if err != nil && strings.Contains(string(output), "system:anonymous") {
+		log.Debug(errors.Wrap(err, "error running command"))
+		log.Debugf("Output is: %v", output)
+		return false
+	}
+	return true
+}
+
+func isServerUp() bool {
+
+	ocpath, err := getOcBinary()
+	if err != nil {
+		log.Debug(errors.Wrap(err, "unable to find oc binary"))
+		return false
+	}
+
+	cmd := exec.Command(ocpath, "whoami", "--show-server")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Debug(errors.Wrap(err, "error running command"))
+		return false
+	}
+
+	server := strings.TrimSpace(string(output))
+	u, err := url.Parse(server)
+	if err != nil {
+		log.Debug(errors.Wrap(err, "unable to parse url"))
+		return false
+	}
+
+	var connectionError error
+	for i := 1; i < 3; i++ {
```

Comment:
I'm not sure if it makes sense to have retries here. Especially if they are directly one after another.

`ocdev` is command line tool that is designed for users to run it manually. I don't think we need retries here.
If there is an error, user will see it, fix it, and run it again.
If it would be daemon or long-running process than it would make sense to have retries.

Reply:
makes sense, removed the loop